ui/public/checkBrowser.js

```diff
@@ -40,7 +40,7 @@ function checkBrowser() {
     d.getElementsByTagName('a')[0].onclick = function () {
       d.getElementsByTagName('div')[0].style.top = '-60px'
     }
-    document.body.prepend(d)
+    document.body.prepend ? document.body.prepend(d) : document.body.insertBefore(d, document.body.firstChild)
```

Comment:
I prefer to directly use `document.body.insertBefore(d, document.body.firstChild)`.

We can use 1 PR to resolve the problems of #770 together (will discuss it later).

Reply:
yeah, however, the support for `firstChild` in chromes <=85 is unknown as caniuse shows

Reply:
It doesn't matter, according to this doc: https://developer.mozilla.org/zh-CN/docs/Web/API/Node/insertBefore , even if firstChild exist or not, insertBefore will work fine.

Reply:
how about `childNodes[0]`.

but i highly doubt Chrome 85- *not* supporting `firstChild` which is in DOM 1 released back in 1997.

Reply:
just curious about why support is unknown. maybe there is a special reason?

Reply:
I think it is fine, according to this doc: https://developer.mozilla.org/zh-CN/docs/Web/API/ParentNode/prepend , `prepend` polyfill is implemented by `insertBefore` and `firstChild`, even if `document.body.firstChild` is undefined, it works as expected, or we can consider using `appendChild`?

Reply:
`unknow` comes from no real data yet. see https://github.com/mdn/browser-compat-data/issues/6369
so yeah we can use `insertBefore` directly